docs/migration-guides/external-providers-primitives-v2.mdx

```diff
@@ -0,0 +1,256 @@
+---
+title: Migrate provider interfaces from backend.run to primitives
+description: Migrate to the primitives interface in external providers
+
+---
+
+# Migrate third-party provider interfaces from `backend.run` to primitives
+
+## Why should third-party providers implement primitives?
+
+The evolution of [user access patterns for QPUs](/migration-guides/qiskit-runtime#why-migrate-to-qiskit-runtime-primitives)
+has led to the Qiskit primitives (`Sampler` and `Estimator`) superseding the `backend.run` interface.
+
+After a period of overlap between `backend.run` and primitives, during which users have gradually transitioned to the 
+primitives access model, the `qiskit-ibm-runtime` provider has discontinued access to IBM&reg; QPUs through `backend.run`.
+
+This guide is for you if the following applies:
+
+- You are in charge of a third-party provider
+- Your interface is still based on `backend.run`
+- You are looking for guidance to implement your own native `Sampler` and `Estimator` primitives
+
+Once you have migrated your interface to primitives, your users with primitives-based code can seamlessly transition between providers.
+
+
+## Exposing custom `Sampler` and `Estimator` implementations
```

Comment:
What do you think about changing "custom implementation" to "native implementation"? _Custom_ feels like it's non-standard, but we are asking the providers to implement primitives because they are the new standard.